### DISCUSSION TITLE
0 0 false line when when rendering a quarto website

### DISCUSSION THREAD

**Discussion by ManuelSpinola** [quarto-dev/quarto-cli#2650] [Q&A]
Dear list members,

When a I render a website from a quarto website I got this two "0 0 false" lines:

<img width="1464" alt="image" src="https://user-images.githubusercontent.com/22131335/192837818-6b82e4ae-3c59-492e-9437-9d7b2ff6edc6.png">

Is there any solution to this?

**Comment by dragonstyle** [comment]
I'm guessing that the listing on this page is collecting unexpected entries and 'rendering' those. Is there any chance you can share the project files/folders? I'd love to see what is fooling us and try to prevent this....

**Comment by ManuelSpinola** [comment]
Thank you very much!

How can I share with you my project files?

Manuel

On Wed, 28 Sep 2022 at 11:58 Charles Teague ***@***.***>
wrote:

> I'm guessing that the listing on this page is collecting unexpected
> entries and 'rendering' those. Is there any chance you can share the
> project files/folders? I'd love to see what is fooling us and try to
> prevent this....
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarto-dev/quarto-cli/discussions/2650#discussioncomment-3755684>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFI3FB3M63J6TZAW5Y7HB73WASBMHANCNFSM6AAAAAAQX7LJSQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
*Manuel Spínola, Ph.D.*
Instituto Internacional en Conservación y Manejo de Vida Silvestre
Universidad Nacional
Apartado 1350-3000
Heredia
COSTA RICA
***@***.*** ***@***.***>
***@***.***
Teléfono: (506) 8706 - 4662
Institutional website: ICOMVIS
<http://www.icomvis.una.ac.cr/index.php/manuel>
Blog sobre Ciencia de Datos: https://mspinola-ciencia-de-datos.netlify.app

**Reply by dragonstyle** [reply]
I think you could attach a zip or point me at a public repo and I'll clone that...

**Comment by ManuelSpinola** [comment]
Thank you very much Charles.

The project is too large and is taking too much time to compress and I
don't have a repo, but I am sending the folder that is making the problem.

If I drop this post (folder) in the blog project everything works fine.

Let me know if that is enough, if not I will compress the entire project.

Manuel
 Archivo.zip
<https://drive.google.com/file/d/16pHkhDrMLAMbKWzzj2J6bichse69Sn5i/view?usp=drive_web>

El mié, 28 sept 2022 a las 14:19, Charles Teague ***@***.***>)
escribió:

> I think you could attach a zip or point me at a public repo and I'll clone
> that...
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarto-dev/quarto-cli/discussions/2650#discussioncomment-3756639>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFI3FB4JPTO7AKJ3N3QP3SLWASR5BANCNFSM6AAAAAAQX7LJSQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Manuel Spínola, Ph.D.*
Instituto Internacional en Conservación y Manejo de Vida Silvestre
Universidad Nacional
Apartado 1350-3000
Heredia
COSTA RICA
***@***.*** ***@***.***>
***@***.***
Teléfono: (506) 8706 - 4662
Institutional website: ICOMVIS
<http://www.icomvis.una.ac.cr/index.php/manuel>
Blog sobre Ciencia de Datos: https://mspinola-ciencia-de-datos.netlify.app

**Reply by dragonstyle** [reply]
Thanks for sharing this - it all looks fine. I think the issue is likely to arise from your however your listing is defined. Can you share the yaml that you have defined on this main / index page?

**Comment by ManuelSpinola** [comment]
Attached.



El vie, 30 sept 2022 a las 8:20, Charles Teague ***@***.***>)
escribió:

> Thanks for sharing this - it all looks fine. I think the issue is likely
> to arise from your however your listing is defined. Can you share the yaml
> that you have defined on this main / index page?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarto-dev/quarto-cli/discussions/2650#discussioncomment-3772879>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFI3FB5XJF24NDNENMWDPLTWA3ZJJANCNFSM6AAAAAAQX7LJSQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Manuel Spínola, Ph.D.*
Instituto Internacional en Conservación y Manejo de Vida Silvestre
Universidad Nacional
Apartado 1350-3000
Heredia
COSTA RICA
***@***.*** ***@***.***>
***@***.***
Teléfono: (506) 8706 - 4662
Institutional website: ICOMVIS
<http://www.icomvis.una.ac.cr/index.php/manuel>
Blog sobre Ciencia de Datos: https://mspinola-ciencia-de-datos.netlify.app

**Reply by dragonstyle** [reply]
Unfortunately I can't see the attachment, could you post the listing QMD contents here on Github?

**Comment by ManuelSpinola** [comment]
project:
  type: website


website:
  title: "Blog sobre Ciencia de Datos"
  site-url: https://mspinola-ciencia-de-datos.netlify.app
  navbar:
    right:
      - ciencia_de_datos.qmd
      - text: "Sobre el autor"
        href: sobre_mi.qmd
      - icon: github
        href: https://github.com/
format:
  html:
    theme: cosmo
    css: styles.css

editor: visual
lang: es



El vie, 30 sept 2022 a las 9:02, Charles Teague ***@***.***>)
escribió:

> Unfortunately I can't see the attachment, could you post the listing QMD
> contents here on Github?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarto-dev/quarto-cli/discussions/2650#discussioncomment-3773196>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFI3FB5LPCNYYRIIIHYCELDWA36JNANCNFSM6AAAAAAQX7LJSQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Manuel Spínola, Ph.D.*
Instituto Internacional en Conservación y Manejo de Vida Silvestre
Universidad Nacional
Apartado 1350-3000
Heredia
COSTA RICA
***@***.*** ***@***.***>
***@***.***
Teléfono: (506) 8706 - 4662
Institutional website: ICOMVIS
<http://www.icomvis.una.ac.cr/index.php/manuel>
Blog sobre Ciencia de Datos: https://mspinola-ciencia-de-datos.netlify.app

**Reply by dragonstyle** [reply]
Thank you for this. Rather than the project / `_quarto.yml`, could you share the listing page itself (it will be a QMD file with a `listing` defined in the front matter)? It would normally be the `index.qmd` file if you created a blog.

**Comment by ManuelSpinola** [comment]
ok.

Like this:

title: "Blog sobre Ciencia de Datos"
listing:
  contents: posts
  sort: "date desc"
  type: default
  categories: true
  sort-ui: false
  filter-ui: false
page-layout: full
title-block-banner: true

This is the list of posts.  Everything was working fine until I added the
last post.

[image: image.png]

El vie, 30 sept 2022 a las 9:20, Charles Teague ***@***.***>)
escribió:

> Thank you for this. Rather than the project / _quarto.yml, could you
> share the listing page itself (it will be a QMD file with a listing
> defined in the front matter)? It would normally be the index.qmd file if
> you created a blog.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarto-dev/quarto-cli/discussions/2650#discussioncomment-3773352>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFI3FBYMZHWIDWJ5OAOMMLTWA4ANBANCNFSM6AAAAAAQX7LJSQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Manuel Spínola, Ph.D.*
Instituto Internacional en Conservación y Manejo de Vida Silvestre
Universidad Nacional
Apartado 1350-3000
Heredia
COSTA RICA
***@***.*** ***@***.***>
***@***.***
Teléfono: (506) 8706 - 4662
Institutional website: ICOMVIS
<http://www.icomvis.una.ac.cr/index.php/manuel>
Blog sobre Ciencia de Datos: https://mspinola-ciencia-de-datos.netlify.app

**Reply by dragonstyle** [reply]
OK great, the issue was exactly with that post as you predicted. If you look inside the post directories, the `datos` contains a cr_shape.qmd file which is matching the listing contents (which is any qmd in the posts folder). That file contains xml data, so the listing is including the XML contents as the 'description' of the listing (which is why the `0 0 false` is making its way onto the page). If you view source you can confirm that the XML is being place into the listing.

- I've made a chance to listings to exclude qmd files which contain no front matter. This should generally fix the problem and will be available in the next pre-release build.
- In addition, if you'd like in the meantime you could change the listing contents to be `posts/*/*.qmd` which should only render qmd files in the direct subfolders of posts.

Thanks for your help tracking this down!

**Comment by ManuelSpinola** [comment]
Thank you very much Charles.

Things got messed up with an ESRI shapefile and its extensions.  I can
delete this type of files, so the problem is easy to fix.

Manuel

El vie, 30 sept 2022 a las 9:58, Charles Teague ***@***.***>)
escribió:

> OK great, the issue was exactly with that post as you predicted. If you
> look inside the post directories, the datos contains a cr_shape.qmd file
> which is matching the listing contents (which is any qmd in the posts
> folder). That file contains xml data, so the listing is including the XML
> contents as the 'description' of the listing (which is why the 0 0 false
> is making its way onto the page). If you view source you can confirm that
> the XML is being place into the listing.
>
>    - I've made a chance to listings to exclude qmd files which contain no
>    front matter. This should generally fix the problem and will be available
>    in the next pre-release build.
>    - In addition, if you'd like in the meantime you could change the
>    listing contents to be posts/*/*.qmd which should only render qmd
>    files in the direct subfolders of posts.
>
> Thanks for your help tracking this down!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/quarto-dev/quarto-cli/discussions/2650#discussioncomment-3773629>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AFI3FBZY2TYLGWZXO4LYPHTWA4EZHANCNFSM6AAAAAAQX7LJSQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
*Manuel Spínola, Ph.D.*
Instituto Internacional en Conservación y Manejo de Vida Silvestre
Universidad Nacional
Apartado 1350-3000
Heredia
COSTA RICA
***@***.*** ***@***.***>
***@***.***
Teléfono: (506) 8706 - 4662
Institutional website: ICOMVIS
<http://www.icomvis.una.ac.cr/index.php/manuel>
Blog sobre Ciencia de Datos: https://mspinola-ciencia-de-datos.netlify.app
